golos.referral/include/golos.referral/config.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <common/config.hpp>
+
+namespace golos { namespace config {
+
+static const uint64_t max_expire = 10000;
+static const uint64_t max_perсent = 8000;  // 80.00%
```

Comment:
While max_expire & max_percent here, and min_breakout & max_breakout in another file?